# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: toolkit/library/rust/shared/Cargo.toml
# Commit: bc5b6dcb7057
# Full Hash: bc5b6dcb7057cf610efe29e14b65e9c3aa34d1c3
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2020-10-12 15:27:19
# Description:
#   Bug 1669162 - Update mapped_hyph to 0.4.2 so that .dic parse errors are non-fatal. r=heycam
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D92435
# ==============================================================================

diff -r a083e04138aa -r bc5b6dcb7057 toolkit/library/rust/shared/Cargo.toml
--- a/toolkit/library/rust/shared/Cargo.toml	Mon Oct 12 09:52:22 2020 +0000
+++ b/toolkit/library/rust/shared/Cargo.toml	Mon Oct 12 10:23:27 2020 +0000
@@ -46,7 +46,7 @@
 neqo_glue = { path = "../../../../netwerk/socket/neqo_glue" }
 rlbox_lucet_sandbox = { version = "0.1.0", optional = true }
 wgpu_bindings = { path = "../../../../gfx/wgpu_bindings", optional = true }
-mapped_hyph = { git = "https://github.com/jfkthame/mapped_hyph.git", tag = "v0.4.0" }
+mapped_hyph = { git = "https://github.com/jfkthame/mapped_hyph.git", tag = "v0.4.2" }
 remote = { path = "../../../../remote", optional = true }
 fog_control = { path = "../../../components/glean", optional = true }
 app_services_logger = { path = "../../../../services/common/app_services_logger" }
